views/components/application-card/examples.njk

```diff
@@ -0,0 +1,89 @@
+{% from "components/styleguide.njk" import sgExample %}
+{% from "./macro.njk" import applicationCard with context %}
+
+{% call sgExample('In progress application') %}
+    {{ applicationCard({
+        type: "pending",
+        projectName: "Scout hut extension for The Scouts of Sutton Coldfield",
+        amountRequested: "£8,500",
+        overview: [{
+            label: "Project dates",
+            value: "2 December, 2020–2 December, 2020"
+        }, {
+            label: "Location",
+            value: "Sutton Coldfield"
+        }, {
+            label: "Organisation",
+            value: "The Scouts of Sutton Coldfield"
+        }],
+        progress: {
+          sectionsComplete: 2,
```

Comment:
Could this be calculated inside the macro rather than having to (presumably) do it manually every time we use this component? eg. we could add a filter function for templates which returns this total?